docs/spec/sources.md

```diff
@@ -330,11 +338,16 @@ type SourceStatus struct {
     // Source.
     // +optional
     SinkURI *apis.URL `json:"sinkUri,omitempty"`
+
+    // CloudEventAttributes are the specific attributes that the Source uses
+    // as part of its CloudEvents.
```

Comment:
I'm not sure I understand what you mean by "attributes that the source *uses* as part of its CloudEvents" . Do you mean "generates/emits" ?

Reply:
Sorry that's just copy&paste from the existing code. The source uses these attributes when emitting events?

Reply:
I does sound ambiguous, but like Lionel said, it's copied from the code directly. Maybe it can be clarified in a separate PR.

Reply:
marking as resolved as I would like this PR to be only about the new CE attribute.